src/core-services/files/xforms/id.js

```diff
@@ -1,6 +1,16 @@
-import { decodeOpaqueIdForNamespace, encodeOpaqueId } from "../../../xforms/id.js";
+import decodeOpaqueIdForNamespace from "@reactioncommerce/api-utils/decodeOpaqueIdForNamespace.js";
+import encodeOpaqueId from "@reactioncommerce/api-utils/encodeOpaqueId.js";
```

Comment:
Should we rename one of these two so they are the same?

`decodeOpaqueIdForNamespace` and `encodeOpaqueIdForNamespace`

or 

`decodeOpaqueId` and `encodeOpaqueId`